com.ibm.wala.cast/build.gradle

```diff
@@ -82,6 +82,9 @@ model {
 						.find { it.targetPlatform.name == targetPlatform.name }
 						.sharedLibraryLinkFile.parent
 					linker.args "-Wl,-rpath,$libDir"
+					if (targetPlatform.operatingSystem.name == "osx") {
```

Comment:
It would be better to use a single-quoted string here (`'osx'`) instead of a double-quoted string (`"osx"`), since no `$` interpolation is needed.